src/spaces/rec2100-hlg.js

```diff
@@ -11,7 +11,7 @@ const scale = 3.7743;	// Place 18% grey at HLG 0.38, so media white at 0.75
 
 export default new RGBColorSpace({
 	id: "rec2100hlg",
-	cssId: "rec2100-hlg",
+	cssId: "--rec2100-hlg",
```

Comment:
This should be undashed https://drafts.csswg.org/css-color-hdr/#valdef-color-rec2100-hlg